backend/event/models.py

```diff
@@ -29,7 +29,23 @@ class EpistolaryEvent(models.Model):
     note = models.TextField(
         null=False,
         blank=True,
-        help_text="Additional notes that describe the event and what connects the letter actions it comprises."
+        help_text="Additional notes that describe the event and what connects the letter actions it comprises.",
+    )
+
+    triggered_world_events = models.ManyToManyField(
+        to="WorldEvent",
+        through="EpistolaryEventTrigger",
+        symmetrical=False,
+        related_name="world_event_triggers",
+        help_text="World events triggered by this epistolary event",
+    )
+
+    triggered_epistolary_events = models.ManyToManyField(
+        to="self",
+        through="EpistolaryEventSelfTrigger",
+        through_fields=("triggering_epistolary_event", "triggered_epistolary_event"),
```

Comment:
Good to know (in case you didn't know this -- I didn't!):
- Without `symmetrical=False`, Django will assume that any model's ManyToMany relationship with itself is symmetrical and will add _two_ records in the database.
- `through_fields` makes sure that when you call `world_event.triggered_world_events.add(...<another-world-event>...)`, this `another-world-event` is mapped to the correct field in the through model. In other words, it marks which of the two foreign fields in the through model is the _source_ and which is the _target_.